### PR TITLE
[skip ci] DNM: daemon: add support for iscsi scenario

### PR DESCRIPTION
**Pull request by leseb**
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

File: src/daemon/entrypoint.sh.in [hunk 1]
```diff
@@ -154,6 +154,33 @@ case "$CEPH_DAEMON" in
     source disk_list.sh
     start_disk_list
     ;;
+  tcmu_runner)
+    # TAG: tcmu_runner
+    if is_redhat; then
+      source start_tcmu_runner.sh
+      start_tcmu_runner
+    else
+      log "ERROR: tcmu_runner scenario is only available on CentOS."
```
Comment (ErwanAliasr1) [inline src/daemon/entrypoint.sh.in]: But also on redhat right ?

File: src/daemon/entrypoint.sh.in [hunk 2]
```diff
@@ -154,6 +154,33 @@ case "$CEPH_DAEMON" in
...
+    fi
+    ;;
+  iscsi_api)
+    # TAG: iscsi_api
+    if is_redhat; then
+      source start_rbd_target_api.sh
+      start_rbd_target_api
+    else
+      log "ERROR: iscsi_api scenario is only available on CentOS."
```
Comment (ErwanAliasr1) [inline src/daemon/entrypoint.sh.in]: Ditto.

File: src/daemon/entrypoint.sh.in (hunk 1 above)
Comment (leseb) [reply src/daemon/entrypoint.sh.in]: Yes, let me clarify.

**Comment by ErwanAliasr1** [comment]
It would be lovely if the commit message explains why/how does only centos/luminous is having iscsi

**Pull request merged**